book/src/simple-payment-and-state-verification.md

```diff
@@ -0,0 +1,158 @@
+# Simple Payment and State Verification
+
+Thin clients using Solana require a receipt for transactions that have been
+accepted by the cluster without running a full validator node.  New fullnode
+validators that desire to boot from a known checkpoint in the network require
+some proof that the checkpoint is valid.
+
+Solana receipts provide a proof that a transaction or state have been confirmed
+by the desired set of validators to a desired amount of finality.
+
+## Trusted Validators
+
+Thin clients send an RPC request to a validator to check for a recent signature
+confirmation.  The validator responds with a confirmation count for the
+signature, and clients are able to query the validator for the error code for
+the signature.  This kind of payment or state verification depends on a trusted
+validators and is not a proof.
+
+## Receipts
+
+The basic idea for receipts is creating a chain from the transaction to a slice
+of PoH ledger that proves that the desired set of validators have voted on a
+block contain that transaction to a desired degree of finality.
+
+Clients can examine this piece of the ledger and compute the locktower lockout
+rules for each of the votes.
+
+The receipts for both, state and payments, start with a merkle path from the
```

Comment:
Every instance of "merkle" should be capitalized. https://en.wikipedia.org/wiki/Ralph_Merkle

Reply:
what about merkle-path or merkle-root?